src/components/ha-badge.ts

```diff
@@ -0,0 +1,155 @@
+import { css, CSSResultGroup, html, LitElement, nothing } from "lit";
+import { customElement, property } from "lit/decorators";
+import { classMap } from "lit/directives/class-map";
+import { ifDefined } from "lit/directives/if-defined";
+import "./ha-ripple";
+
+type BadgeType = "badge" | "button";
+
+@customElement("ha-badge")
```

Comment:
So now we'll have `hui-badge` and `ha-badge` 😬 Feels a bit chaotic. 

Should we rename `hui-badge` to something that represents it's creating one from config?

Reply:
We created `hui-card` and `hui-badge` because we had `hui-view` 😅. I agree, it's super confusing.